tools/src/test/java/org/apache/kafka/tools/MetadataQuorumCommandErrorTest.java

```diff
@@ -53,12 +53,12 @@ public void testDescribeOptions() {
 
     @Test
     public void testRelativeTimeMs() {
-        long nowMs = Instant.now().toEpochMilli();
-        assertTrue(MetadataQuorumCommand.relativeTimeMs(nowMs, "test") >= 0);
-        long invalidEpochMs = Instant.EPOCH.minus(1, ChronoUnit.DAYS).toEpochMilli();
+        long validEpochMs = Instant.now().minusSeconds(5).toEpochMilli();
+        assertTrue(MetadataQuorumCommand.relativeTimeMs(validEpochMs, "test") >= 0);
+        long invalidEpochMs = Instant.EPOCH.minusSeconds(86400).toEpochMilli();
         assertThrows(KafkaException.class, () -> MetadataQuorumCommand.relativeTimeMs(invalidEpochMs, "test"));
-        long futureTimestampMs = Instant.now().plus(1, ChronoUnit.DAYS).toEpochMilli();
-        assertThrows(KafkaException.class, () -> MetadataQuorumCommand.relativeTimeMs(futureTimestampMs, "test"));
+        long futureEpochMs = Instant.now().plus(1, ChronoUnit.DAYS).toEpochMilli();
+        assertThrows(KafkaException.class, () -> MetadataQuorumCommand.relativeTimeMs(futureEpochMs, "test"));
```

Comment:
I wonder if this is the right way to fix it. My understanding is that `relativeTimeMs` fails if the provided timestamp is not strictly greater than the current time. Should we loosen this validation to accept a timestamp greater than or equals to the current time?

Reply:
Yeah, I was thinking the same. The check now is `epoch < timestamp < now`, but it should be `epoch < timestamp <= now`. In the last commit I fixed `relativeTimeMs` and added back the `nowMs` check. Thanks for the review.